docs/usage.rst

```diff
@@ -188,27 +188,24 @@ Because processing this emerging type of datasets (*densely sampled neuroimaging
 
 Troubleshooting
 ---------------
-Logs and crashfiles are outputted into the
+Logs and crashfiles are output into the
 ``<output dir>/fmriprep/sub-<participant_label>/log`` directory.
-Information on how to customize and understand these files can be found on the
-`nipype debugging <http://nipype.readthedocs.io/en/latest/users/debug.html>`_
-page.
```

Comment:
This page has disappeared from recent documentation. Not sure an equivalent page is available, I have therefore discarded the whole sentence.

Reply:
https://miykael.github.io/nipype_tutorial/notebooks/basic_debug.html

Reply:
Done.